### PR TITLE
BE | Create `GoodDeedsController#create` Endpoint

### PR DESCRIPTION
**Pull request by MelTravelz**
# Changes 
[ ] Bug fix
[X] New feature
[X] This change requires a documentation update

## List of Changes: 
- This PR is related to this issue: #13 
- Create GoodDeed model & spec
        - Happy Path & Sad Path tests
        - Create #add_participants instance method in Good Deeds model
- Create Users::GoodDeedsController & create action
- Good Deeds Request Spec
        - Happy Path, Sad Path, & Edge case tests
- Create GoodDeedsSerializer

### CODE REVIEW

**Comment by KaraJoHo** [comment]
Merge conflicts resolved!

File: app/controllers/api/v1/users/good_deeds_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# app/controllers/api/v1/users/good_deeds_controller.rb
+class Api::V1::Users::GoodDeedsController < ApplicationController
+  def create
+    new_good_deed = GoodDeed.new(name: params[:deed_name], host_id: params[:user_id], date: params[:date], time: params[:time])
+    if new_good_deed.save!
```
Comment (KaraJoHo) [inline app/controllers/api/v1/users/good_deeds_controller.rb:7]: Very nice! Thank you for taking this on, it looks great
